Makefile

```diff
@@ -176,6 +180,9 @@ client: ${PREFIX}/bin/notary
 binaries: ${PREFIX}/bin/notary-server ${PREFIX}/bin/notary ${PREFIX}/bin/notary-signer
 	@echo "+ $@"
 
+escrow: ${PREFIX}/bin/escrow
```

Comment:
@cyli I think once we have configurable integration with the NotaryRepository that would make sense. At the moment it doesn't really plug into anything and until @n4ss's dependency injection PR is merged, there's no way to plug it in short of forked code.